DOCS.md

```diff
@@ -0,0 +1,61 @@
+# Documentation best practices
+This doc is still work in progress and will constantly be updated.
+
+1. Keep the examples in the stories minimal, without using unnecessary props that don't serve the purpose of demonstrating the story unless the Vibe's designers specifically request it.
+2. Whenever possible, avoid adding titles to the variants displayed within the story to maintain clear and simpler story code.
+3. When laying out components, utilize Vibe's Flex component instead of implementing your own flex layout using CSS. Refer to the code example below for clarification.
+```js
+<Flex direction={Flex.directions.COLUMN} gap={Flex.gaps.MEDIUM}>
+          <Steps type={Steps.types.NUMBERS} steps={steps} activeStepIndex={2} />
+          <Steps type={Steps.types.GALLERY} steps={steps} activeStepIndex={2} />
+          <Steps
+            steps={steps}
+            type={Steps.types.GALLERY}
+            activeStepIndex={2}
+            areNavigationButtonsHidden
+            className="monday-storybook-steps_padding"
+          />
+</Flex>
+```
+
+4. If you need to add titles to the component states displayed in a story, make use of the VisualDescription component, as shown in the code example provided.
+```js
+<Flex direction={Flex.directions.ROW} gap={Flex.gaps.SMALL}>
```

Comment:
1. If possible, keep shorter examples that focus on the StoryDescription. One is enough.
2. remove classNames
3. maybe worth adding an image to explain what is the outcome of this

Reply:
I removed most of the props which not required for the example and keep only 2 StoryDescription usages (because of it is a single component I prefer they will not use the StoryDescription for this)